servers/lib/config/libms.yaml.default

```diff
@@ -1,12 +1,10 @@
 port: '4001'
-mode: 'git'
+mode: 'local'
 local-path: '..\..\files'
 log-level: 'debug'
 apollo-path: '/lib'
 graphql-playground: 'true'
 
 git-repos:
   - user-1: 
-      repo-url: 'https://github.com/isomorphic-git/lightning-fs'
-  - user-2: 
-      repo-url: 'https://github.com/isomorphic-git/lightning-fs'
\ No newline at end of file
+      repo-url: 'https://github.com/INTO-CPS-Association/DTaaS.git'
```

Comment:
`repo-url: 'https://gitlab.com/dtaas/user1.git'`

Reply:
@nichlaes please use the following yaml for `libms.yml.default`

```yaml
port: '4001'
mode: 'local'    #git or local
local-path: '..\..\files'
log-level: 'debug'
apollo-path: '/lib'
graphql-playground: 'true'

git-repos:   #only used in git mode
  - user1: 
      repo-url: 'https://gitlab.com/dtaas/user1.git'
  - user2: 
      repo-url: 'https://gitlab.com/dtaas/user2.git'
  - common: 
      repo-url: 'https://gitlab.com/dtaas/common.git'
```

Reply:
It makes sense to use the same in README.md as well. We are using convention of **user1, user2, common** etc. So being consistent in configuration makes sense as well.

Reply:
please add the same structure to `config/libms.dev.yml`